### ISSUE TITLE
Unable to download parity via Homebrew due to hyperium/hyper git update error

### ISSUE THREAD

**Issue by Melvillian** [ethcore/parity#1053]
I am trying to install parity using Homebrew and running into the following error

```
Alexs-MacBook-Pro-3:parity alex$> bash <(curl https://get.parity.io -Lk)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   154  100   154    0     0    158      0 --:--:-- --:--:-- --:--:--   169
100   154  100   154    0     0     83      0  0:00:01  0:00:01 --:--:--   956
100 12858  100 12858    0     0   5228      0  0:00:02  0:00:02 --:--:-- 7921k
==> Checking OS dependencies
 ✓	 OS X Version 10.11.4
 ✓	 git
 ✓	 Ruby 2.0.0
 ✓	 Homebrew 0.9.9 (git revision d887; last commit 2016-05-06)
Homebrew/homebrew-core (git revision 79038; last commit 2016-05-06)

Found all dependencies (4/4)
==> OK, let's install Parity now!
==> Last chance! Sure you want to install this software? [Y/n] Y

==> Installing Parity build dependencies
==> Adding ethcore repository

==> Updating brew
Already up-to-date.

==> Installing parity
==> Reinstalling ethcore/ethcore/parity
==> Cloning https://github.com/ethcore/parity.git
Updating /Library/Caches/Homebrew/parity--git
==> Checking out branch beta
==> multirust update stable
==> multirust run stable cargo build --release
Last 15 lines from /Users/alex/Library/Logs/Homebrew/parity/02.multirust:
2016-05-06 14:40:43 -0700

multirust run stable cargo build --release

    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/tomusdrw/rust-ctrlc.git`
    Updating git repository `https://github.com/tomusdrw/jsonrpc-http-server.git`
    Updating git repository `https://github.com/hyperium/hyper`
Unable to update https://github.com/hyperium/hyper?branch=mio#11930b61

To learn more, run the command again with --verbose.

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/ethcore/homebrew-ethcore/issues

Error: No such keg: /usr/local/Cellar/parity

==> Parity is missing

==> Installation failed
```

Running with --verbose doesn't reveal any more information about why there is a problem with updating hyperium/hyper. I read in the gitter that there was a recent update to the version of hyper-mio, perhaps that has caused this problem?

**Comment by jacogr** [comment]
The same happens when trying to build from the master branch on OS X -

```
parity git:(master)$ cargo build --release
    Updating git repository `https://github.com/NikVolf/json-ipc-server.git`
    Updating git repository `https://github.com/tomusdrw/jsonrpc-core.git`
    Updating git repository `https://github.com/ethcore/nanomsg.rs.git`
    Updating git repository `https://github.com/hyperium/hyper`
Unable to update https://github.com/hyperium/hyper?branch=mio#11930b61

To learn more, run the command again with --verbose.
```

Then with --verbose

```
parity git:(master)$ cargo build --release --verbose
    Updating git repository `https://github.com/hyperium/hyper`
Unable to update https://github.com/hyperium/hyper?branch=mio#11930b61

Caused by:
  [4/-3] Revspec '11930b61857e15830d558303ba5e77f6c02d06fd' not found.
```

**Comment by NikVolf** [comment]
@jacogr 
can you please try branch `hyper-update-2` ?
like here
https://github.com/ethcore/parity/pull/1054

**Comment by debris** [comment]
fixed in #1054 #1055
